### DISCUSSION TITLE
how to achieve load balancing

### DISCUSSION THREAD

**Discussion by howeWang97** [apache/pulsar#23703] [Q&A]
Ask for help how to achieve load balancing?
I used DotPulsar,I deployed a ZooKeeper server, two brokers, and three bookies on a server. The ports for the two brokers are 16650 and 16652. Creating a producer using 16652 works fine, but creating a producer using 16650 fails and no messages can be sent. I tried closing 16652, and 16650 was able to work, but when 16652 was active, 16650 was unable to work.

**Comment by lhotari** [comment]
@howeWang97 You can get good answers to many questions about Pulsar by using the [Pulsar Slack](https://pulsar.apache.org/community/#section-discussions) #ask-ai channel. Have you tried it already?
